edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/Authenticator.cs

```diff
@@ -26,16 +31,20 @@ public Authenticator(IAuthenticator tokenAuthenticator, string edgeDeviceId, ICr
         /// <summary>
         /// Authenticates the client credentials
         /// </summary>
+        /// <param name="clientCredentials">Client credentials</param>
+        /// <returns>true if authentication succeeds, otherwise false.</returns>
         public Task<bool> AuthenticateAsync(IClientCredentials clientCredentials)
             => this.AuthenticateAsync(clientCredentials, false);
 
         /// <summary>
-        /// Reauthenticates the client credentials
+        /// Re-authenticates the client credentials
         /// </summary>
+        /// <param name="clientCredentials">Client credentials</param>
```

Comment:
The idea of stylecop is nice, but I have used it before, and I feel some its restrictions are very annoying during development - like requiring public methods to have comments. I know you can turn it off during development, but that can be intrusive as well. 
We should discuss this first before pushing the change in, I have some reservations about it